### PR TITLE
MDS-115 [ADD] Reimpliment Logging Class

### PR DESCRIPTION
**Pull request by justinelinette**
# [MDS-115](https://jacobfholland.atlassian.net/browse/MDS-115)
### Description: 
Reimplement logging class

### Implementation: 
None

### CODE REVIEW

File: medusa/__init__.py [hunk 1]
```diff
@@ -43,6 +43,10 @@ def create_app(env="development"):
         # Install colored logging.
         coloredlogs.install()
 
+        # check for logs folder and create if not found
+        if not os.path.exists("logs"):
+            os.makedirs("logs")
```
Comment (jacobfholland) [inline medusa/__init__.py:48]: Are we sure we want to include `logs` directory directly in the `medusa` folder? Should it not live outside of `medusa` where our `scripts` folder already is? This is a design decision and your call. Though I think it should be outside the medusa package.

**Review by jacobfholland — CHANGES_REQUESTED** [review]
I have given it a review. Really great job for totally doing it on your own. I have marked some requested changes for you.

File: medusa/config/app.py [hunk 2]
```diff
@@ -19,7 +19,8 @@ class Config(DatabaseConfig):
     DATABASE_NAME = os.environ.get("DATABASE_NAME", "medusa")
     FLASK_APP = os.environ.get("FLASK_APP", "run.py")
     SECRET_KEY = os.environ.get("SECRET_KEY", secrets.token_bytes(32))
-    DEBUG = False
+    DEBUG = True
+    LOG_PATH = "logs/mds.log"
```
[the remark this answers is not in the document]
Comment (justinelinette) [reply medusa/config/app.py:23]: log path split into log name, log path, log file

File: medusa/log/log.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    def builder(self, message, status):
+        model = inspect.stack()[
+            2][0].f_locals["self"].__class__.__name__.upper()
+        function = inspect.stack()[2][0].f_code.co_name.upper()
+        return f"[{model}-{function}: {status}] {message}"
+
+    # TODO: debug outputs to log file, but not console
+    def debug(self, message, status):
+        built_message = self.builder(message, status)
+        self.logger.debug(built_message)
```
[the remark this answers is not in the document]
Comment (justinelinette) [reply medusa/log/log.py:27]: made these changes for all logging methods

**Review by jacobfholland — APPROVED** [review]

**Pull request merged**